client/src/plugins/camunda-plugin/start-instance-tool/StartInstanceTool.js

```diff
@@ -179,6 +179,12 @@ export default class StartInstanceTool extends PureComponent {
        !deployService.canDeployWithConfiguration(deploymentConfig) ||
        await this.checkConnection(deploymentConfig.endpoint);
 
+     this.setState({ overlayState: {
+       handleClose: () => {
+         deployService.closeOverlay();
+       }
+     } });
```

Comment:
Due to missing context: Can you explain why we need to set such a close handler to the state?